src/SelfServe/SelfServeLoadingComponentAdapter.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * This adapter is responsible to render the React component
+ * If the component signals a change through the callback passed in the properties, it must render the React component when appropriate
+ * and update any knockout observables passed from the parent.
+ */
+import * as ko from "knockout";
+import { Spinner, SpinnerSize } from "office-ui-fabric-react";
+import * as React from "react";
+import { ReactAdapter } from "../Bindings/ReactBindingHandler";
+
+export class SelfServeLoadingComponentAdapter implements ReactAdapter {
+  public parameters: ko.Observable<number>;
+
+  constructor() {
+    this.parameters = ko.observable(Date.now());
+  }
+
+  public renderComponent(): JSX.Element {
+    return <Spinner size={SpinnerSize.large} />;
```

Comment:
I check for the logic to render it on Main.tsx
This is only rendered when selfServetype is undefined.
I cant move it to SelfServeComponent or the adapter because this needs to be displayed even if only the explorer is going to be displayed (and not selfServeTypes's UI)